MOBILE/CellCamSetDistance.md

```diff
@@ -0,0 +1,18 @@
+---
+ns: MOBILE
+aliases: ["0x53F4892D18EC90A4"]
+---
+## CELL_CAM_SET_DISTANCE
+
+```c
+// 0x53F4892D18EC90A4
+void CELL_CAM_SET_DISTANCE(float p0);
+```
+
+```
```

Comment:
Please don't use code blocks for descriptions.
Also like Disquse said, name should be prefixed with `_`.